roles/cs.varnish/templates/vcl/subroutines/recv_throttling.vcl.j2

```diff
@@ -9,6 +9,7 @@
         ### Throttling rule - {{ rule.id }} ###
```

Comment:
This implementation is not bad, it looks like it should work fine, but I think this code is starting to get messy.
I think we could generate single if condition containing all conditions that should be present,
somethink like this:
```
    {%- set conditions = [] -%}
    {%- if rule.path_pattern | default(false, true) -%}
        {%- set _ = conditions.append('req.url ~ "' ~ rule.path_pattern ~ '"') -%}
    {%- endif -%}
    {%- if rule.user_agent | default(false, true) -%}
        {%- set _ = conditions.append('req.http.User-Agent ~ "' ~ rule.user_agent ~ '"') -%}
    {%- endif -%}
    {%- if rule.method | default(false, true) -%}
        {%- set _ = conditions.append('req.method == "' ~ rule.method | upper ~ '"') -%}
    {%- endif -%}
```

we should also consider adding check (in tasks) for case where none of conditions are present (no path, user_agent and method, because that would lead to empty condition)
    if ({{ conditions | join(' && ') }}) {```


We do not need user-agent in `X-RateLimit-Identity` - because this is only needed for getting unique limit and we have to keep in mind that it's send back to the client - we should not forward back original user agent

we could instead add flag that would ignore user ip (default false) that would gate if we should have `req.http.x-real-ip` added to identity